src/main/java/edu/cornell/kfs/paymentworks/PaymentWorksConstants.java

```diff
@@ -18,28 +18,60 @@
  */
 package edu.cornell.kfs.paymentworks;
 
+import org.apache.commons.lang.StringUtils;
+
+import edu.cornell.kfs.fp.CuFPConstants.ScheduledSourceAccountingLineConstants.ScheduleTypes;
+
 public class PaymentWorksConstants {
+	
+	public static final String PAYMENT_WORKS_FIELD_MAPPING_DOCUMENT_TYPE = "PMWM";
 
-	public static final String PAYMENT_WORKS_VENDOR_SEQ = "PAYMENT_WORKS_VENDOR_SEQ";
+	public static final String PAYMENT_WORKS_VENDOR_SQ = "PAYMENT_WORKS_VENDOR_SQ";
+	public static final String PAYMENT_WORKS_FIELD_MAPPING_SQ = "PAYMENT_WORKS_FIELD_MAPPING_SQ";
```

Comment:
the field mapping sequence name was too long with the full SEQ ... I thought it would be better for the two to be consistent.  Also, before this work, the database objects didn't exist.  Would you prefer SEQ to SQ